utils/PolicyManager.py

```diff
@@ -44,16 +44,20 @@ def is_enabled(self, alert_type):
 
     def get_parameters(self, alert_type):
         try:
-            for item in self.active_policy.items:
-                if item.alert_type_code == alert_type:
-                    parameters = json.loads(item.alert_type.parameters)
-                    parameters = {par : val['default'] for par, val in parameters.items()}
+            for i in range(0,2):
```

Comment:
The algorithm flow is a little confusing, It is not easy to follow. I think it would be simpler if it splitted in three parts:
1- Look for the item
2- If it is not found, create the item and save it.
2- Take the parameters of the item and return them

Reply:
Done